### PR TITLE
Adapt header as the figma prototype

### PR DESCRIPTION
**Pull request by oltionchampari**
Closes Caleydo/tdp_bi_bioinfodb#1212

### CODE REVIEW

File: src/initialize.app.ts [hunk 1]
```diff
@@ -10,11 +10,7 @@ import * as aboutDisclaimer from './templates/_aboutDisclaimer.html';
 import {Ordino} from 'ordino';
 import {AppMetaDataUtils} from 'phovea_ui';
 
-const _ = new Ordino({
-  showCookieDisclaimer: true,
-  showResearchDisclaimer: false,
```
Comment (thinkh) [inline src/initialize.app.ts:15]: We must keep this options.

File: src/initialize.app.ts [hunk 2]
```diff
@@ -10,11 +10,7 @@ import * as aboutDisclaimer from './templates/_aboutDisclaimer.html';
...
-  showAboutLink
-});
+const _ = new Ordino();
 
 function showAboutLink(title: HTMLElement, content: HTMLElement) {
```
Comment (thinkh) [inline src/initialize.app.ts:15]: If you remove the `showAboutLink` option above, you can remove this function.

**Review by thinkh — CHANGES_REQUESTED** [review]
The header still looks the same as before:

grafik

File: src/initialize.app.ts (hunk 1 above)
Comment (oltionchampari) [reply src/initialize.app.ts]: You are right

File: src/initialize.app.ts (hunk 2 above)
Comment (oltionchampari) [reply src/initialize.app.ts]: Also correct, thanks

**Comment by oltionchampari** [comment]
> The header still looks the same as before:
> 
> grafik
> 
> Does this PR require [Caleydo/ordino#294](https://github.com/Caleydo/ordino/pull/294)?

It does require Caleydo/ordino#294, failed to write it.

**Review by thinkh — APPROVED** [review]
Ok, looks fine in combination with https://github.com/Caleydo/ordino/pull/294/

**Pull request merged**